web/components/Icons/DownChevron.tsx

```diff
@@ -0,0 +1,10 @@
+const DownChevron : any = () => {
```

Comment:
For future reference --

You can avoid using `any` and no error will be thrown since type inference will occur (I think). If not a function expression will surely do it, and is more common with react.